latest.py

```diff
@@ -50,10 +52,8 @@ def goto(target_x, target_y, reverse):
         drivetrain.drive_for(REVERSE, distance, MM, wait=True)
     else:
         drivetrain.drive_for(FORWARD, distance, MM, wait=True)
-
 def pick_up():
     fork_motor_group.spin_to_position(1500, DEGREES, wait=True)
-
 def set_down():
     fork_motor_group.spin_to_position(1800, DEGREES, wait=True)
```

Comment:
60 more points (from 70 to 130) with the same 60 lines by removing empty lines?

Reply:
That's the goal

Reply:
That's the goal